### DISCUSSION TITLE
How do you view what is currently transfering in the background?

### DISCUSSION THREAD

**Discussion by timothyparez** [mucommander/mucommander#1076] [Q&A] [answered]
If you `F6`, `TAB`, `Enter` you copy from source to location and hide the UI.  
But then there seems to be no way to view what's transferring in the background.  
There's even no indication (that I can tell of) that it's transferring anything at all.  
  
I resort to using the `Activity Monitor` in `Ubuntu` to check if there is still stuff being transferred  
from my computer to my network drive.  

Am I missing something?

**Comment by pskowronek** [comment, marked as answer]
![obraz](https://github.com/mucommander/mucommander/assets/360665/6e4f01ed-95c3-4a96-bfed-970faa500699)

However, I see that if file already exists and overwrite option is set to `Ask`, the popup is not shown until you switch between other app window and return to muC (can you check this, if so, pls create a separate bug for it).

**Reply by timothyparez** [reply]
I never noticed that icon before. Thank you.
I will check the overwrite issue.
